bin/build.sh

```diff
@@ -22,12 +22,14 @@
 
 # Set these flags to control various installation options
 INSTALL_DEPENDENCIES=1
+INSTALL_RUST=1
 BUILD_Z3=1
 BUILD_BOOGIE=1
 BUILD_CORRAL=1
 BUILD_SYMBOOGLIX=1
 BUILD_LOCKPWN=1
 BUILD_SMACK=1
+TEST_SMACK=0
```

Comment:
This should not be changed. Fix this.

Reply:
I replaced the intro to build.sh with the current content in develop.